### PR TITLE
feat: webinar page redesign

### PR DESCRIPTION
**Pull request by simboonlong**
### pr updates
- webinar banner replace
- replace contents in webinar page

### user story
- https://www.pivotaltracker.com/story/show/173847745
- https://www.pivotaltracker.com/story/show/173847756

### preview
Screenshot 2020-07-28 at 2 22 00 PM
Screenshot 2020-07-28 at 2 22 29 PM

### CODE REVIEW

File: src/components/UI/AnnoucementBar/AnnoucementBar.tsx [hunk 1]
```diff
@@ -16,15 +16,7 @@ export const AnnoucementBarUnStyled = ({ className }: AnnoucementBarProps) => {
...
-                  <h1 className="mb-3">TradeTrust Tech Webinar Series</h1>
-                  <h5 className="mb-3">
-                    Advance your knowledge with our captivating talks, interactive content and more for free!
-                  </h5>
-                  <p className="mb-0">
-                    This series of tech talks is organised by the Infocomm Media Development Authority of Singapore
-                    (IMDA) and GovTech Singapore. It comprises six webinars and aims to provide professionals with
-                    knowledge on TradeTrust as a digital utility for cross border trade.
-                  </p>
+                  <img className="banner-title" src="/static/images/webinar/banner-title.png" alt="" />
```
Comment (yehjxraymond) [inline src/components/UI/AnnoucementBar/AnnoucementBar.tsx]: if you want to have alt, can fill in the alt, empty alt looks awkward lol

File: src/components/WebinarPageContent/SectionDetails.tsx [hunk 2]
```diff
@@ -11,111 +11,275 @@ export const SectionDetailsUnStyled = ({ className }: SectionDetailsProps) => {
...
+              <div className="row">
+                <div className="col-12">
+                  <p>
+                    This non-technical session helps provide a foundational and critical understanding of TradeTrust as
+                    a digital utility as well as the mental framing necessary as a pre-requsite for subsequent webinars.
+                  </p>
+                </div>
+              </div>
             </div>
           </div>
```
Comment (yehjxraymond) [inline src/components/WebinarPageContent/SectionDetails.tsx]: Each of these look like a repeatable block, do you want to have it rendered from data instead?

Comment (simboonlong) [reply src/components/WebinarPageContent/SectionDetails.tsx]: yup sure

**Review by yehjxraymond — APPROVED** [review]
lgtm

**Pull request merged**